Release v0.8.0

invidian opened the pull request:
Closes #49

invidian commented on CHANGELOG.md:
```diff
@@ -0,0 +1,57 @@
+- Made operator code compatible with Kubernetes 1.17.
+- Fixed publishing leader acquired event by the operator.
+
+[0.8.0]: https://github.com/flexkube/libflexkube/compare/v0.7.3...v0.8.0
```
Oops, nice catch :smile: I was copy pasting from my repo :)

tormath1 replied:
I assumed but I did not find this lines from the `flexkube/libflexkube` changelog so I was wondering. :)

tormath1 reviewed (approved):
LGTM - it would be nice to have the issues / PR link in the generated changelog for the next releases :) Let's ship it :rocket:

invidian commented:
Fixed couple of typos and grammar mistakes in the changelog, also added mention about rename of Go module.

>it would be nice to have the issues / PR link in the generated changelog for the next releases :)

Thanks for review and suggestion @tormath1 :wink: 

Right now changelog is hand-crafted :hammer_and_wrench: and many items mentioned are spread over several PRs and commits, so it would be quite time consuming to find them all, but indeed, it would be nice to have it automated. I guess "changelog" line on PRs would be good, the same as Kubernetes does.

invidian commented:
If there is no objections, I'll merge and make RC release. I'd also like to automate pushing of container images before we release.

tormath1 commented:
@invidian let's go for merging this. Images pushing can be done in a follow-up PR.

> I guess "changelog" line on PRs would be good, the same as Kubernetes does.

This is also how we do on the mantle side, we recently added a changelog too. It's currently done manually because it's not that "hard" to maintain, you just need to think about adding your entry when you submit a PR. (https://github.com/flatcar-linux/mantle/blob/flatcar-master/CHANGELOG.md#unreleased)
